controller/src/settings/ui.rs

```diff
@@ -695,6 +704,134 @@ impl SettingsUI {
         ui.text("Weapon!");
     }
 
+    fn render_esp_settings_bomb(
+        &mut self,
+        settings: &mut AppSettings,
+        ui: &imgui::Ui,
+        target: EspSelector,
+    ) {
+        let config_key = target.config_key();
+        let config_enabled = settings
+            .esp_settings_enabled
+            .get(&config_key)
+            .cloned()
+            .unwrap_or_default();
+
+        let config = match settings.esp_settings.entry(config_key.clone()) {
+            Entry::Occupied(entry) => {
+                let value = entry.into_mut();
+                if let EspConfig::Bomb(value) = value {
+                    value
+                } else {
+                    log::warn!("Unable to find bomb");
+                    *value = EspConfig::Bomb(EspBombSettings::new(&target));
+                    if let EspConfig::Bomb(value) = value {
+                        value
+                    } else {
+                        unreachable!()
+                    }
+                }
+            }
+            Entry::Vacant(entry) => {
+                if let EspConfig::Bomb(value) =
+                    entry.insert(EspConfig::Bomb(EspBombSettings::new(&target)))
+                {
+                    value
+                } else {
+                    unreachable!()
+                }
+            }
+        };
+        let _ui_enable_token = ui.begin_enabled(config_enabled);
```

Comment:
Maybe we can dedup all this ugly code just for the two tab bars 🤔

Reply:
yeah, i did it in like 3 minutes just pressing ctrl c + v lol